kubo-rpc/src/pin.rs

```diff
@@ -24,12 +21,8 @@ pub async fn remove<T>(_client: T, ipfs_path: &IpfsPath) -> Result<Cid, Error>
 where
     T: IpfsDep,
 {
-    if let Some(cid) = ipfs_path.cid() {
-        // Beetle does not have any garbage collection for its store so everything is pinned.
-        // Therefore we do not need to track which blocks are pinned.
-        // Do nothing
-        Ok(*cid)
-    } else {
-        Err(Error::Invalid(anyhow!("IPFS path does not have a CID")))
```

Comment:
This error isn't possible anymore?

Reply:
Correct, previously the code was using the `IpfsPath` type from Iroh that was either an path rooted in a CID or a path rooted in an IPNS name. The new code now uses an `IpfsPath` implemented within the crate that only allows the IPFS (cid) use case.